projects/plugins/boost/app/assets/src/js/features/page-cache/meta/meta.tsx

```diff
@@ -210,3 +209,40 @@ const BypassPatterns = ( {
 		</div>
 	);
 };
+
+const BypassPatternsExample = () => {
+	const [ show, setShow ] = useState( false );
+
+	return (
+		<div className={ styles[ 'example-wrapper' ] }>
+			{ /* eslint-disable-next-line jsx-a11y/anchor-is-valid */ }
+			<a
+				href="#"
+				className={ styles[ 'example-button' ] }
+				onClick={ e => {
+					e.preventDefault();
+					setShow( ! show );
+				} }
+			>
+				{ __( 'See an example', 'jetpack-boost' ) }
+			</a>
```

Comment:
It should use the children prop passed to the component so that whatever text was passed while using this component remains.
```suggestion
const BypassPatternsExample = ( { children } ) => {
	const [ show, setShow ] = useState( false );

	return (
		<div className={ styles[ 'example-wrapper' ] }>
			{ /* eslint-disable-next-line jsx-a11y/anchor-is-valid */ }
			<a
				href="#"
				className={ styles[ 'example-button' ] }
				onClick={ e => {
					e.preventDefault();
					setShow( ! show );
				} }
			>
				{ children }
			</a>
```

Reply:
Nice catch! Though the linter was a tad unhappy so I had to modify your suggestion in fd40ad6.